fastapi_jwks/models/types.py

```diff
@@ -25,3 +25,66 @@ class JWKSConfig(BaseModel):
 class JWTTokenInjectorConfig(BaseModel):
     payload_field: str = "payload"
     token_field: str = "raw_token"
+
+
+class JWTHeader(BaseModel):
```

Comment:
Maybe you can add link to the RFCs, they really help trying to understand the models